docs/src/constants/redirect-routes.js

```diff
@@ -69,7 +69,6 @@ export default {
   '/components/native/navigation-bar': ROUTES.NAVIGATION_BAR,
   '/components/native/navigation-stack': ROUTES.NAVIGATION_STACK,
   '/components/native/nudger': ROUTES.NUDGER,
-  '/components/native/pagination-dots': ROUTES.PAGINATION_DOT,
```

Comment:
Why's this been removed?

Reply:
`ROUTES.PAGINATION_DOT` does not exist. https://backpack.github.io/components/native/pagination-dots/ is just a broken page

Reply:
Dayum